.github/workflows/publish_bundle.yaml

```diff
@@ -16,12 +16,11 @@ jobs:
     steps:
       - name: checkout code
         uses: actions/checkout@v4
-      # Note the use of the upload-charm action. Bundles can be treated as charms.
       - name: Upload Bundle
-        uses: canonical/charming-actions/upload-charm@main
+        uses: canonical/charming-actions/upload-bundle@main
         with:
             credentials: "${{ secrets.CHARMCRAFT_AUTH }}"
-            charm-path: ./bundle
+            bundle-path: ./bundle
             github-token: "${{ secrets.GITHUB_TOKEN }}"
             channel: "k8s/edge"
             tag-prefix: "bundle"
```

Comment:
Just check if this option is supported on the upload-bundle action

Reply:
Good point. I've already checked the source of the action and, though undocumented, but it is supported.